### DISCUSSION TITLE
Moving from OpenKeychain suggestion(gpg-agent)

### DISCUSSION THREAD

**Discussion by plant7403** [zeapo/Android-Password-Store#1676] [Ideas]
With all these moving away from openkeychain
Is there a chance that aps would act as gpg agent for other apps, and have "encrypt" button 😁
Like a capability to store other keys in the password manager, and interact with some
Thank you

**Comment by msfjarvis** [comment]
I do not intend to create a worse OpenKeychain, no.

**Reply by plant7403** [reply]
Maybe just storing files/other keys? Just for secure storage of them? 😅

**Reply by msfjarvis** [reply]
You can store as many GPG keys as your heart desires, and you can stuff whatever files you want into your password store repository. APS will not offer anything on top of that.
